index.html

```diff
@@ -2,45 +2,265 @@
+          </div>
         </div>
-        <div class="space"></div>
-      </section>
-    </main>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
```

Comment:
- Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly